Tests/Shared/XCTestCase+MockChatChannels.swift

```diff
@@ -0,0 +1,51 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import XCTest
+
+extension XCTestCase {
+    /// Mocked array of `ChatChannel` data.
+    func chatChannelsMock() -> [ChatChannel] {
```

Comment:
nit: Maybe this can rather be an extension on `[ChatChannel]`? You can also use `dummy()` for the name to match our LLC naming.

Reply:
Sure, refactored.